sdk/metricsadvisor/ai-metrics-advisor/test/public/advisorclient.spec.ts

```diff
@@ -520,7 +519,7 @@ matrix([[true, false]] as const, async (useAad) => {
 
       describe("Feedback", async function () {
         let createdFeedbackId: string;
-        it.skip("creates Anomaly feedback", async function () {
+        (useAad ? it.skip : it)("creates Anomaly feedback", async function () {
```

Comment:
How about we skip the whole feedback describe block?  could it be achieved by wrapping with a `if (!useAad) { describe("Feedback".....) }`?